worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -163,6 +166,7 @@ private MemoryManager(CelebornConf conf) {
                 logger.info("Trigger action: RESUME PUSH and REPLICATE");
                 memoryPressureListeners.forEach(
                     memoryPressureListener -> memoryPressureListener.onResume("all"));
+                pausePushDataTime.addAndGet(System.currentTimeMillis() - lastPauseTime);
```

Comment:
the state machine does not look correct to me.

Reply:
`RESUM` always be triggered after `PAUSE_PUSH`

Reply:
what if the worker goes into PUSH_AND_REPLICATE_PAUSED?